app.rb

```diff
@@ -72,4 +75,74 @@ def create_teacher(age, name, specialty)
     puts 'Teacher created successfully'
     puts
   end
+
+  def load_files
+    @books = load_books
+    load_persons
+    load_rentals(@persons, @books)
+  end
```

Comment:
- [ ] Great job saving the JSON files 👍 however loading the books is not working as expected, which is partly because the `load_files` method is never called.

Reply:
Fixed